plugins/inputs/lustre2_lctl/lustre2_lctl.go

```diff
@@ -0,0 +1,44 @@
+//go:build linux
+
+//go:generate ../../../tools/readme_config_includer/generator
+package lustre2_lctl
+
+import (
+	_ "embed"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+const namespace = "lustre2"
+
+//go:embed sample.conf
+var sampleConfig string
+
+// Lustre proc files can change between versions, so we want to future-proof
+// by letting people choose what to look at.
+type Lustre2Lctl struct {
+	OstCollect    []string        `toml:"ost_collect"`
+	MdtCollect    []string        `toml:"mdt_collect"`
+	ClientCollect []string        `toml:"client_collect"`
+	Log           telegraf.Logger `toml:"-"`
+}
+
+func (*Lustre2Lctl) SampleConfig() string {
+	return sampleConfig
+}
+
```

Comment:
I would like to see an `Init` function in here that ensures that the user has access to the `lctl` command before we start up and fail right after we try to gather.